slsa-provenance/generator/cmd/main.go

```diff
@@ -48,34 +49,35 @@ func main() {
 	if z == nil || len(z) != 2 {
 		log.Fatal(errors.New("sha256 digest is not valid"))
 	}
-	org := z[0]
-	repo := z[1]
+	// org := z[0]
+	// repo := z[1]
 
-	// make github client
-	token, ok := os.LookupEnv("GITHUB_TOKEN")
-	if !ok {
-		log.Fatal(errors.New("missing GITHUB_TOKEN"))
-	}
-	ctx := context.Background()
-	// Requires a token with repo scope
-	ts := oauth2.StaticTokenSource(
-		&oauth2.Token{AccessToken: token},
-	)
-	tc := oauth2.NewClient(ctx, ts)
-	client := github.NewClient(tc)
+	// // make github client
+	// token, ok := os.LookupEnv("GITHUB_TOKEN")
+	// if !ok {
+	// 	log.Fatal(errors.New("missing GITHUB_TOKEN"))
+	// }
+	// ctx := context.Background()
+	// // Requires a token with repo scope
+	// ts := oauth2.StaticTokenSource(
+	// 	&oauth2.Token{AccessToken: token},
+	// )
+	// tc := oauth2.NewClient(ctx, ts)
+	// client := github.NewClient(tc)
 
-	workflow, run, job, err := provenance.GetCurrentWorkflowRunAndBuildJob(ctx, client, org, repo, ghRunId)
-	if err != nil {
-		log.Fatal(err)
-	}
+	// workflow, run, job, err := provenance.GetCurrentWorkflowRunAndBuildJob(ctx, client, org, repo, ghRunId)
+	// if err != nil {
+	// 	log.Fatal(err)
+	// }
 
-	att, err := provenance.GenerateAttestation(workflow, run, job, digest)
-	if err != nil {
-		log.Fatal(err)
-	}
+	// att, err := provenance.GenerateAttestation(workflow, run, job, digest)
+	// if err != nil {
+	// 	log.Fatal(err)
+	// }
+	att := "{my-into-attestation}"
```

Comment:
is this purposely skipped because the run id was commented out so it can't find the workflow run?